tests/store/artifact/test_databricks_artifact_repo.py

```diff
@@ -1186,3 +1190,220 @@ def test_log_artifacts_provides_failure_info(self, databricks_artifact_repo, tmp
             assert "MOCK ERROR 1" in err_msg
             assert "file_2.txt" in err_msg
             assert "MOCK ERROR 2" in err_msg
+
+
+def test_multipart_upload(databricks_artifact_repo, tmp_path):
+    size = 250 * 1000 * 1000  # 250 MB -> requires 3 part uploads
+    large_file = tmp_path.joinpath("large_file")
+    with large_file.open("wb") as f:
+        f.seek(size - 1)
+        f.write(b"\0")
```

Comment:
Can we write various bytes data to file, and in following assertion checking the uploading data in request ?

Reply:
@WeichenXu123 Can you write code?

Reply:
e.g , we can write file content like `aaa...bbb...ccc...`, then in following assertion we can check the request containing each part data is `aaa...`, `bbb...`, `ccc...`

Reply:
@harupy @WeichenXu123 I agree, it would be nice to verify that we're correctly uploading the data in chunks (feel free to mock a smaller chunk size for testing). Can we add an assert on the data passed to `http_request_mock`?